### PR TITLE
OCPBUGS-39118: Update Lightspeed logo to new standards

### PR DESCRIPTION
**Pull request by rhamilto**
After

light
dark

### CODE REVIEW

File: frontend/packages/console-app/src/components/lightspeed/Lightspeed.scss [hunk 1]
```diff
@@ -74,28 +75,24 @@
...
-
-    &::after {
-      border-color: var(--pf-v5-global--palette--white) !important;
-    }
+    background-image: url('logo-dark.svg');
+    background-color: var(--pf-v5-global--BackgroundColor--100) !important;
```
Comment (rhamilto) [inline frontend/packages/console-app/src/components/lightspeed/Lightspeed.scss:83]: FYI @kyoto

**Comment by jhadvig** [comment]
```
[36mINFO[0m[2024-09-06T08:29:29Z] Ran for 55m43s                               
[31mERRO[0m[2024-09-06T08:29:29Z] Some steps failed:                           
[31mERRO[0m[2024-09-06T08:29:29Z] 
  * could not run steps: step backend failed: test "backend" failed: could not watch pod: the pod ci-op-6r3kjxcc/backend failed after 3m40s (failed containers: sidecar, test): ContainerFailed one or more containers exited
```

/retest

**Pull request merged**